framework/wazuh/tests/data/schema_global_test.sql

```diff
@@ -139,6 +139,16 @@ INSERT INTO agent (id, name, ip, register_ip, internal_key, os_name, os_version,
                    'node01',strftime('%s','now','-4 days'),
                     strftime('%s','now','-12 seconds'),'updated','active','group-2,group-1');
 
+-- Connected agent in group-1

```

Comment:
Description?
In my opinion, use `agent with different OS` or something similar.